### DISCUSSION TITLE
怎么实现全局热键关键词搜索

### DISCUSSION THREAD

**Discussion by iamansen** [shuax/MouseInc.Settings#307] [Q&A]
比如选中关键字按快捷键直接跳转到网页搜索，而不是弹出菜单后在选择

**Comment by Ctrl-Ling** [comment]
动作示例里面不就有吗。。
![image](https://user-images.githubusercontent.com/36662875/178178995-8effac02-3b8d-4c15-b82a-c646a21cfab4.png)
